### DISCUSSION TITLE
AirFlow Integration [metrics] statsd_datadog_enabled

### DISCUSSION THREAD

**Discussion by nicolamarangoni** [DataDog/datadog-agent#13799] [Q&A]
This manual looks obsolete: https://docs.datadoghq.com/integrations/airflow/?tab=host
I'm currently using the following setting:
```
[metrics]
statsd_datadog_enabled = True
```
and it looks like
```
[scheduler]
statsd_on = True
```
isn't needed anymore, as the metrics are sent to the DogStatsD running per default when the agent is started (on `localhost:8125`).
Furthermore, the datadog.yaml with the specific settings for AirFlow isn't needed anymore.
However, I don't find any documentation about the parameter `statsd_datadog_enabled`.
The AirFlow documentation doesn't offer any detail and the is no mention of it in the DataDog docs.

**Comment by eugeneYWang** [comment]
I saw your post on airflow repo as well. Did you figure it out?

**Comment by kzap** [comment]
DataDog has this PR in their documentation recommending to set it to False https://github.com/DataDog/integrations-core/pull/16378
